quickstart/backend/src/main/java/com/digitalasset/quickstart/service/LicenseApiImpl.java

```diff
@@ -562,10 +283,25 @@ private static ValueOuterClass.Identifier parseTemplateIdentifier(String templat
         }
         String entityName = entityNameBuilder.toString();
 
-        return ValueOuterClass.Identifier.newBuilder()
-                .setPackageId(packageId)
-                .setModuleName(moduleName)
-                .setEntityName(entityName)
-                .build();
+        return ValueOuterClass.Identifier.newBuilder().setPackageId(packageId).setModuleName(moduleName)
+                .setEntityName(entityName).build();
+    }
+
+    private AnyValue toAnyValueContractId(String contractId) {
+        return new AnyValue.AnyValue_AV_ContractId(new ContractId<>(contractId));
+    }
+
+    private static RelTime parseRelTime(String durationStr) {
+        Duration duration = Duration.parse(durationStr);
+        long micros = duration.toNanos() / 1_000;
+        return new RelTime(micros);
+    }
+
+    private static OffsetDateTime toOffsetDateTime(Instant instant) {
+        return instant == null ? null : OffsetDateTime.ofInstant(instant, ZoneOffset.UTC);
+    }
```

Comment:
These don't look awfully specific to the licence API implementation. `toAnyValueContractId` also looks like it can be `static`.